### DISCUSSION TITLE
Remove chat signatures by default

### DISCUSSION THREAD

**Discussion by mrrpmrreeoow** [PurpurMC/Purpur#1024] [__Dismissed__] [answered]
This prevents reporting players, by default it will be on false it can be something as `use-signatures: false` (in `purpur.yml`).
I think this will help a lot of servers.

**Comment by katekerllenevich** [comment, marked as answer]
We don't ever change something from the server "by default".

We also aren't going to be adding this option, there are plugins that can do it. We don't want to disable a function that protects children from pedophiles and catches terrorists in the act.

**Reply by mrrpmrreeoow** [reply]
> We don't ever change something from the server "by default".
> 
> We also aren't going to be adding this option, there are plugins that can do it. We don't want to disable a function that protects children from pedophiles and catches terrorists in the act.

Thank you for the reply : ) Im happy, that we can use plugins for that (paper will make these plugins not working)

**Reply by katekerllenevich** [reply]
> paper will make these plugins not working

Do you have a source for this?

**Reply by sueriol88** [reply]
> > paper will make these plugins not working
> 
> Do you have a source for this?

not sure if there is a source, but I'm curious if Paper does in fact break those plugins will Purpur still allow them to work? or should that be discussed in a separate discussion when it becomes relevant. thanks

**Reply by katekerllenevich** [reply]
Paper doesn't break plugins for no reason. At this point its just fearmongering and alarmism bordering on a conspiracy theory.
